src/cascadia/TerminalApp/defaults.json

```diff
@@ -219,6 +234,8 @@
         { "command": "find", "keys": "ctrl+shift+f" },
 
         // Tab Management
+        // "command": "closeTab" is unbound by default.

```

Comment:
Should we bind this to something? This seems like basic functionality we should have a binding for.

Reply:
The windows "standard" is ctrl+f4, but be aware that it doesn't ask for confirmation and _will_ destroy any number of panes worth of work.

Reply:
I mean, we don't ask for confirmation when you click the 'x' button on the tab now.

Reply:
You're right. I'm not saying it's _more_ dangerous than the `X` button, just that it's _as_ dangerous. If we litter minefields around folks' keyboards, like we once did with <kbd>ctrl+w</kbd>, there's a chance for DSAT. Not a huge one, just a chance.

Reply:
I'm personally ok with it being unbound be default. Mainly because there's other methods to close a tab (i.e.: `exit`, click X button, close pane, etc...)

Reply:
I'd prefer it to be bound to `ctrl+f4`. Every other command except `unbound` has a default binding.

Reply:
@cinnamon-msft and I had an offline discussion. We'll be holding off on ctrl+f4 being bound to close tab until we can get a confirmation dialog for closing the tab, if multiple panes are open.

follow-up work item: #XXXX